lib/vmpooler/pool_manager.rb

```diff
@@ -138,6 +129,14 @@ def _check_ready_vm(vm, pool, ttl, provider)
         end
       end
 
+      $redis.hset('vmpooler__vm__' + vm, 'check', Time.now)
```

Comment:
In my mess of rebasing and squashing I've managed to make it look like this change is part of my change, but this was introduced in #239